subworkflows/local/amp.nf

```diff
@@ -65,8 +74,40 @@ workflow AMP {
 
         AMP_HMMER_HMMSEARCH ( ch_in_for_amp_hmmsearch )
         ch_versions = ch_versions.mix(AMP_HMMER_HMMSEARCH.out.versions)
+//        GUNZIP_HMMER ( AMP_HMMER_HMMSEARCH.out.output )
```

Comment:
Plesae remove for now rather tha ncommenting out, or at least say _why_ it's commented out :grimace:

Reply:
Alright, I added a comment instead of deleting it. So that we can use it for a future issue